modules/core/shared/src/main/scala/lucuma/core/enums/F2Filter.scala

```diff
@@ -11,25 +11,15 @@ import lucuma.core.util.Enumerated
  * Enumerated type for Flamingos2 filters.
  * @group Enumerations (Generated)
  */
-enum F2Filter(
-  val tag: String,
-  val shortName: String,
-  val longName: String,
-  val wavelength: Option[Wavelength],
-  val obsolete: Boolean
-) derives Enumerated:
+enum F2Filter(val tag: String, val shortName: String, val longName: String, val wavelength: Option[Wavelength], val obsolete: Boolean) derives Enumerated:
 
   case Y      extends F2Filter("Y",      "Y",       "Y (1.02 um)",        Wavelength.fromIntPicometers(1020000), false)
-  case F1056  extends F2Filter("F1056",  "F1056",   "F1056 (1.056 um)",   Wavelength.fromIntPicometers(1056000), false)
```

Comment:
These are the important changes, the rest is fluff

Reply:
I got rid of the `obsolete` flags when I removed the obsolete GMOS stuff on the theory that it isn't sufficient to manage resources with a boolean flag.  Maybe we need it for something though?